packages-exp/app-exp/package.json

```diff
@@ -15,19 +15,21 @@
     "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "build": "rollup -c && yarn api-report",
-    "build:release": "rollup -c rollup.config.release.js && node ./use_public_typings.js",
+    "build:release": "rollup -c rollup.config.release.js && yarn typings:public",
```

Comment:
Can you please add `yarn api-report` as the second step? It generates the rollup d.ts files.

Reply:
Done!